src/main/java/com/aerotrack/utils/clients/ryanair/RyanairClient.java

```diff
@@ -24,26 +28,111 @@
 public class RyanairClient {
 
     private final RyanairApiService ryanairApiService;
-    private final static String RYANAIR_COOKIE = "rid.sig=jyna6R42wntYgoTpqvxHMK7H+KyM6xLed+9I3KsvYZaVt7P36AL6zp9dGFPu5uVxaIiFpNXrszr+LfNCdY3IT3oCSYLeNv/ujtjsDqOzkY66AL3V6kH2vsK+au12X21HkZ4S8GaG8CoBmm/m0rLsOKYkxtw+U3+ejBaPc15jJjKXnc3owMBg82SNbqyKjVd6Z6qcsoE25p3RmlcaHuHC3GBf1yIGtlqeQun3Mj0vmSURVPQBjK65pge2zGBymoORV6PsmZ0Nabv73gS2VkhG+Eccz5iSIRPrZm5cloi/941TFo8oiXdyqvTMx/ozox2fkvaKB2vd/goSx543TxPdKoGKRLaDY3FoIepe6I46UFvXEZYszzugXHYRnp0lbIn/HPyvHH/iW/TXRrqsELQabKd1hH+Ut1ZgpfsKEtwDVyL7mVvi1qEOqHddSVKCN/439KxQqi9K03dQDm+knQaLRpzZL8EYqCeSaosMOeEhc2CAYWLW2D5jH5iTot0YiyaN3QJFE59H3MYDpGjoZfTfen83yVSpT8DBahOOr6Eibv62bKQXBxel5kIm75dZB26iUzmkBs1Iags291UJ8wpu/GtBD1rghlZoRJt9u3ASkAj3P85dBcV8MwGykVWJ4mCO; mkt=/gb/en/; .AspNetCore.Session=CfDJ8NJy2CjeBXdEiTlIkEo9jP1x6eP6igWNkoeL9uCto1qdz97HQLRlCAJWIhw97YY5uemEBnTrcLNnoB7lqKOnEJRzF%2F4yhGKN9A5COUteaQmZduO6o2whfYqdyD1Qd%2B%2BH6EXjFP4cd0DtvPDwguugs%2Bc684C2CSfw16lyvYFtSkvU; fr-correlation-id=b8d2a2fe-383b-44fa-8c4a-ba6cab8c994d; rid=034ed121-51ec-4cac-9196-97a50ee42d2c; RY_COOKIE_CONSENT=true; STORAGE_PREFERENCES={\"STRICTLY_NECESSARY\":true,\"PERFORMANCE\":false,\"FUNCTIONAL\":false,\"TARGETING\":false,\"SOCIAL_MEDIA\":false,\"PIXEL\":false,\"GANALYTICS\":true,\"__VERSION\":2}; myRyanairID=";
-
     public static RyanairClient create() {
         Retrofit retrofit = new Retrofit.Builder()
-                .baseUrl("https://www.ryanair.com/api/booking/v4/it-it/")
+                .baseUrl("https://www.ryanair.com/api/")
                 .addConverterFactory(ScalarsConverterFactory.create())
                 .build();
 
         return new RyanairClient(retrofit.create(RyanairApiService.class));
     }
 
+    public List<String> getAirportConnections(String airportCode) {
+        try {
+            Response<String> response = ryanairApiService.getAirportConnections(airportCode).execute();
+            if(!response.isSuccessful())
+            {
+                log.error("Unsuccessful response from Ryanair API when trying to get airport connections: {}", response);
+                throw new HttpException(response);
+            }
+
+            String responseBody = response.body();
```

Comment:
se response è null questa lancia NullPointerException

Reply:
fai come nell'altro metodo tipo

Reply:
oppure catcha NullPointerException, ma facciamo in un modo solo nel client penso